src/Maestro/tests/Maestro.ScenarioTests/ScenarioTests_Subscriptions.cs

```diff
@@ -0,0 +1,265 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Maestro.ScenarioTests.ObjectHelpers;
+using Microsoft.DotNet.Darc;
+using Microsoft.DotNet.Maestro.Client.Models;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Maestro.ScenarioTests
+{
+    [TestFixture]
+    [Category("PostDeployment")]
+    public class ScenarioTests_Subscriptions : MaestroScenarioTestBase
+    {
+        private TestParameters _parameters;
```

Comment:
If these were a local variable instead of a field, you could avoid the setup/teardown by just having the test start with
```csharp
using var parameters = await TestParemeters.GetAsync();
SetTestParameters(_parameters);
```

Then the class is stateless, and there is no chance any sort of base class shenanigans would break anything. :-)

Reply:
Excellent point. I'll make that change with the next PR. Thanks for the suggestion!